ch_tools/chadmin/internal/zookeeper.py

```diff
@@ -493,7 +493,7 @@ def _get_zk_client(ctx):
         connect_str,
         auth_data=auth_data,
         timeout=timeout,
-        logger=logging.getLogger("chadmin"),
+        logger=logging.getNativeLogger("kazoo"),
```

Comment:
Passing loguru logger to other libraries results in broken formatting due to `%s` usage. This way `InterceptHandler` passes correctly formatted message to the corresponding loguru logger.